src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -418,7 +418,9 @@ public double getBonusBlockCost() {
 	}
 	
 	public double getTownBlockCost() {
-		return (Math.pow(TownySettings.getClaimPriceIncreaseValue(), getTownBlocks().size()) * TownySettings.getClaimPrice());
+		double price=(Math.pow(TownySettings.getClaimPriceIncreaseValue(), getTownBlocks().size()) * TownySettings.getClaimPrice());
+		double maxprice=TownySettings.getMaxClaimPrice();
+		return (maxprice!=-1?Math.min(price, maxprice):price);
```

Comment:
If you could fix the spacing so there are spaces before and after the `=, !=, ?, :` and so forth, that would make this look like the rest of the code.